### PR TITLE
Mark filter objects listener as private

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

File: CHANGELOG-6.0.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ in 6.0 versions.
     - `PRE_FETCH_OBJECTS` => `FOS\ElasticaBundle\Persister\Event\PreFetchObjectsEvent`
 * **[BC break]** Renamed `FOS\ElasticaBundle\Persister\Event\OnExceptionEvent::setIgnore()` to `FOS\ElasticaBundle\Persister\Event\OnExceptionEvent::setIgnored()`.
 * **[BC break]** Marked all `fos_elastica.manager` services as private.
+* **[BC break]** Marked the `fos_elastica.filter_objects_listener` service as private.
```
Comment (XWB) [inline CHANGELOG-6.0.md:48]: This should be moved to beta 2.

Comment (deguif) [reply CHANGELOG-6.0.md]: Fixed

Comment (deguif) [reply CHANGELOG-6.0.md]: Oups conflict, I'm rebasing

Comment (deguif) [reply CHANGELOG-6.0.md]: Done

**Pull request merged**